c2corg_ui/static/js/documentediting.js

```diff
@@ -151,6 +151,12 @@ app.DocumentEditingController = function($scope, $element, $attrs, appLang,
   this.submit = false;
 
   /**
+   * @type {Date}
+   * @export
+   */
+  this.today = new Date();
```

Comment:
Mmmm
I would say it's simpler to create a new controller based upon the outing editing controller and to have ``today`` in the new image editing controller as well (because this property is not needed for other document types).